tests/codex/testvalidation.nim

```diff
@@ -12,19 +14,33 @@ import ./helpers
 asyncchecksuite "validation":
   let period = 10
   let timeout = 5
-  let maxSlots = 100
+  let maxSlots = MaxSlots(100)
+  let validationGroups = ValidationGroups(8).some
   let slot = Slot.example
   let proof = Groth16Proof.example
   let collateral = slot.request.ask.collateral
 
   var validation: Validation
   var market: MockMarket
   var clock: MockClock
+  var groupIndex: uint16
+
+  proc initValidationParams(maxSlots: MaxSlots, validationGroups: ?ValidationGroups, groupIndex: uint16): ValidationParams =
+    without validationParams =? ValidationParams.init(maxSlots, groups=validationGroups, groupIndex), error:
+      raiseAssert fmt"Creating ValidationParams failed! Error msg: {error.msg}"
+    validationParams
+  
+  func createValidation(clock: Clock, market: Market, validationParams: ValidationParams): Validation =
```

Comment:
🟠 Again, just a suggestion, I think the more "nim way" is to prefix a new call with `new`:
```suggestion
  func newValidation(clock: Clock, market: Market, validationParams: ValidationParams): Validation =
```

Reply:
Agree!